company_website/static/common/scripts/set_current_year_in_footer.js

```diff
@@ -0,0 +1,3 @@
+let current_year = new Date().getUTCFullYear().toString()
```

Comment:
Also name of file is wrong. Please fix naming to follow javascript convention

Reply:
What exactly is wrong with that name? :/ Should I use hyphen instead of underline? I see that in project are .js files with names  in both naming conventions.